### PR TITLE
Update documentation for delete_user & $last_seen

### PR DESCRIPTION
**Pull request by markoshust**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -82,6 +82,10 @@ mixpanel.people.clear_charges('billybob');
 // delete a user
 mixpanel.people.delete_user('billybob');
 
+// delete a user in Mixpanel Engage without altering $last_seen
+// - pass option `$ignore_time: true` to prevent the $last_seen property from being updated
+mixpanel.people.delete_user('billybob', {$ignore_time: true});
```
Comment (tdumitrescu) [inline readme.md]: I'm afraid I don't quite get this; can you explain what the intention is? Once a profile is deleted, it doesn't exist anymore, nor does its `last_seen` property.

Comment (markoshust) [inline readme.md]: This is very important if you ever want to re-import the same user. The $last_seen property will see the time of the delete_user event, unless $ignore_time is passed in.

If this isn't passed in, and the user profile is re-imported with the same uid, the $last_seen will see the delete_user event as the last action, and it's irreversible.

Comment (tdumitrescu) [inline readme.md]: Interesting. I don't feel great about relying on an undocumented implementation detail (that a 'delete' action leaves around the association of distinct ID -> last seen); but on the other hand it seems to be the only way to preserve last seen if reimporting old data, so I'll merge this for the time being, and hopefully we can update the ingestion code in the future to behave a little more sanely for the underlying issue (i.e. don't update last seen as part of a delete action).

**Comment by tdumitrescu** [comment]
Also a word of caution: this undocumented behavior will currently only work if you reimport more or less immediately after deletion. Sometime within the subsequent 24 hours deleted profiles should be wiped completely. Makes me think that exposing this in the README is going to cause more trouble than help...

**Comment by Quuxplusone** [comment]
IMHO, Ted's original intuition ("deleted users don't have last-seen times") is correct, and where the current observable behavior differs from that intuition, it's indicative of a backend bug, not a problem with the intuition. You should *not* rely on the last-seen time of a deleted user getting preserved across deletion-and-recreation. As for the bug, now that it's on our radar, I would hope to see it fixed by the end of this coming week. If you see this patch reverted soon, it'll probably indicate that the underlying issue has been fixed. :)

**Comment by markoshust** [comment]
Thanks all. I found this out when talking to a mixpanel support engineer. It is definitely an important bug, as I was trying to wipe & delete user records for a somewhat new account, since I passed in some badly named fields. Since the data is immutable, deleting and re-creating the user records was the only option, and I was unable to do so because I didn't pass in that extra param apparently.

If the patch is reverted, can you confirm back when that happens that it's fixed on backend? Thanks!

**Pull request merged**